fenix/app/src/main/java/org/mozilla/fenix/components/appstate/AppState.kt

```diff
@@ -79,4 +83,22 @@ data class AppState(
     val wallpaperState: WallpaperState = WallpaperState.default,
     val standardSnackbarError: StandardSnackbarError? = null,
     val shoppingState: ShoppingState = ShoppingState(),
+    val screen: Screen = Screen.Home,
```

Comment:
one thought for future consideration here: how will the default value here be updated based on the user's "Opening screen" preference?